modules/viqeoBidAdapter.js

```diff
@@ -0,0 +1,168 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {logError, logInfo, _each, mergeDeep, isFn, isNumber, isPlainObject} from '../src/utils.js'
+import {BANNER} from '../src/mediaTypes.js';
+import {Renderer} from '../src/Renderer.js';
+
+const BIDDER_CODE = 'viqeo';
+const VIQEO_ENDPOINT = 'https://ads.betweendigital.com/openrtb_bid';
+const RENDERER_URL = 'https://cdn.viqeo.tv/js/vq_starter.js';
+const DEFAULT_CURRENCY = 'USD';
+const DEFAULT_SSPID = 44697;
+
+function getBidFloor(bid) {
+  const {floor, currency} = bid.params;
+  const curr = currency || DEFAULT_CURRENCY;
+  if (!isFn(bid.getFloor)) {
+    return {floor: isNumber(floor) ? floor : 0, currency: curr};
+  }
+  const floorInfo = bid.getFloor({currency: curr, mediaType: BANNER, size: '*'});
+  if (isPlainObject(floorInfo) && isNumber(floorInfo.floor) && floorInfo.currency === curr) {
+    return floorInfo;
+  }
+  return {floor: floor || 0, currency: currency || DEFAULT_CURRENCY};
+}
+
+/**
+ * @type {BidderSpec}
+ */
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+  /**
+   * @param {BidRequest} bidRequest The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: ({params}) => {
+    if (!params) {
+      logError('failed validation: params not declared');
+      return false;
+    }
+    if (!params.user && !params.user?.buyeruid) {
+      logError('failed validation: user.buyeruid not declared');
+      return false;
+    }
+    if (!params.playerOptions) {
+      logError('failed validation: playerOptions not declared');
+      return false;
+    }
+    const {profileId, videoId, playerId} = params.playerOptions;
+    if (!profileId) {
+      logError('failed validation: profileId not declared');
+      return false;
+    }
+    if (!videoId && !playerId) {
+      logError('failed validation: videoId or playerId not declared');
+      return false;
+    }
+    return true;
```

Comment:
It would be helpful for you to add a table with the required and optional params available to Publishers in your markdown file. You could copy over the table from your prebid.github.io PR, https://github.com/prebid/prebid.github.io/pull/3987/

Reply:
Fixed it.